src/services/legal-s.ts

```diff
@@ -0,0 +1,32 @@
+/** A legal filing object from the API. */
+export interface LegalFilingI {
+  // contains one arbitrarily-named legal filing output
+  // whose name is the filing type
+  // eg, alteration, annualReport, etc
+  [name: string]: string // link to fetch it
+}
+
+/** Response object from LegalServices.fetchDocuments(). */
+export interface FetchDocumentsI {
```

Comment:
These interfaces aren't being used here, do we need them? Also the comment should be updated since its referencing class api stuff